### ISSUE TITLE
Future Traffic Light State not in tf.Example Format

### ISSUE THREAD

**Issue by chrisyrniu** [waymo-research/waymo-open-dataset#355]
The future state of traffic light is described in tf.Example (https://waymo.com/open/data/motion/tfexample), but when I tried to extract the corresponding information, an error occurred:

"InvalidArgumentError: Key: traffic_light_state/future/x.  Can't parse serialized Example."

Could you please confirm that the future state of traffic light is not provided in tf.Example? Is it possible that we can access it somewhere else? Thank you very much!

**Comment by scott-ettinger** [comment]
The feature 'traffic_light_state/future/x' should exist in the tf_example files. Could you tell me which files you are using when you get this error?

**Comment by chrisyrniu** [comment]
Thanks for the response! I tried again and the error just gone. I cannot remember the file I tried before but I might make some mistakes on setting the dimensions of the future traffic light state.

Other than that, I have problems on getting 'traffic_light_state/past/timestamp_micros', 'traffic_light_state/current/timestamp_micros' and 'traffic_light_state/future/timestamp_micros'. Do they exist in the tf.Example file? Thanks!
